homeassistant/components/caldav/calendar.py

```diff
@@ -130,24 +132,26 @@ def setup_platform(
             name = calendar.name
             device_id = calendar.name
             entity_id = generate_entity_id(ENTITY_ID_FORMAT, device_id, hass=hass)
-            calendar_devices.append(
-                WebDavCalendarEntity(name, calendar, entity_id, days)
+            coordinator = CalDavUpdateCoordinator(
+                hass,
+                calendar=calendar,
+                days=days,
+                include_all_day=False,
+                search=None,
             )
+            calendar_devices.append(WebDavCalendarEntity(name, entity_id, coordinator))
 
     add_entities(calendar_devices, True)
 
 
-class WebDavCalendarEntity(CalendarEntity):
+class WebDavCalendarEntity(
+    CoordinatorEntity["CalDavUpdateCoordinator"], CalendarEntity
```

Comment:
Are the quotes really needed for typing to work?

Reply:
No, it doesn't. In this case, it would make sense to extract the coordinator into an `coordinator.py` (which is kinda the standard nowadays).

Reply:
Agree, its only needed because it was below in the file. It is really just left here to make review easier since the code was already in place. I'll merge this, then move it out in a follow up PR to make changelog/review clear.